Formula/moco.rb

```diff
@@ -1,15 +1,15 @@
 class Moco < Formula
   desc "Stub server with Maven, Gradle, Scala, and shell integration"
   homepage "https://github.com/dreamhead/moco"
-  url "https://search.maven.org/remotecontent?filepath=com/github/dreamhead/moco-runner/0.10.2/moco-runner-0.10.2-standalone.jar"
-  version "0.10.2"
-  sha256 "ef946d090d3108843708c194809c57f192e3623dbbbcf86bdee54ce93c299a41"
+  url "https://repo1.maven.org/maven2/com/github/dreamhead/moco-runner/0.11.0/moco-runner-0.11.0-standalone.jar"
```

Comment:
Can you please retain the original URL and just update the version components in the URL? I believe we want to continue using `https://search.maven.org/` instead of hard-coding a specific server.